frontend/src/libs/components/app/app.tsx

```diff
@@ -16,6 +33,17 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  useEffect(() => {
+    if (!localStorage.getItem(StorageKey.TOKEN)) {
+      navigate(AppRoute.SIGN_IN);
+    }
+    void dispatch(authActions.getUser())
+      .unwrap()
+      .catch(() => {
```

Comment:
Actually, the redirection will navigated on private route abstraction

Reply:
agree, for now we just need to get user